src/api/config/speedsettings.rs

```diff
@@ -209,6 +209,19 @@ impl SpeedSettings {
     speed <= 1
   }
 
+  /// Set default rdo-lookahead-frames for different speed settings
+  pub fn rdo_lookahead_frames(speed: usize) -> usize {
+    if speed == 10 || speed == 9 {
```

Comment:
a [match](https://doc.rust-lang.org/rust-by-example/flow_control/match.html) would be clearer.

Reply:
Done https://github.com/xiph/rav1e/pull/2674/commits/e015505686e81a1d20191c8033a125edb2f84260